Translations of `Translations` component updated

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OSRD/Translations](https://hosted.weblate.org/projects/osrd/home/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/osrd/home/horizontal-auto.svg)

mansil committed:
front: home: add translation using weblate (Portuguese)

Signed-off-by: Manuela Silva <mmsrs@sky.com>

mansil committed:
front: home: translated using weblate (Portuguese)

Currently translated at 1.8% (12 of 636 strings)

Translation: OSRD/Translations
Translate-URL: https://hosted.weblate.org/projects/osrd/home/pt/

Signed-off-by: Manuela Silva <mmsrs@sky.com>

mansil committed:
front: home: translated using weblate (Portuguese)

Currently translated at 2.0% (13 of 636 strings)

Translation: OSRD/Translations
Translate-URL: https://hosted.weblate.org/projects/osrd/home/pt/

Signed-off-by: Manuela Silva <mmsrs@sky.com>

flomonster reviewed (approved):
LGTM (reviewed with deepl)

mansil committed:
front: home: translated using weblate (Portuguese)

Currently translated at 17.7% (113 of 636 strings)

Translation: OSRD/Translations
Translate-URL: https://hosted.weblate.org/projects/osrd/home/pt/

Signed-off-by: Manuela Silva <mmsrs@sky.com>

flomonster committed:
front: home: translated using weblate (Portuguese)

Currently translated at 18.2% (116 of 636 strings)

Translation: OSRD/Translations
Translate-URL: https://hosted.weblate.org/projects/osrd/home/pt/

Signed-off-by: Florian Amsallem <florian.amsallem@gmail.com>
